[2024-02-11T00:08:35.169+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_weather_data.cleanup_temp_files manual__2024-02-11T00:07:16.875125+00:00 [queued]>
[2024-02-11T00:08:35.249+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_weather_data.cleanup_temp_files manual__2024-02-11T00:07:16.875125+00:00 [queued]>
[2024-02-11T00:08:35.257+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2024-02-11T00:08:35.434+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): cleanup_temp_files> on 2024-02-11 00:07:16.875125+00:00
[2024-02-11T00:08:35.473+0000] {standard_task_runner.py:60} INFO - Started process 112 to run task
[2024-02-11T00:08:35.494+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_weather_data', 'cleanup_temp_files', 'manual__2024-02-11T00:07:16.875125+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpmb2z057h']
[2024-02-11T00:08:35.550+0000] {standard_task_runner.py:88} INFO - Job 7: Subtask cleanup_temp_files
[2024-02-11T00:08:36.206+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_weather_data.cleanup_temp_files manual__2024-02-11T00:07:16.875125+00:00 [running]> on host 09659ec0e379
[2024-02-11T00:08:37.896+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Lucas Andrini' AIRFLOW_CTX_DAG_ID='etl_weather_data' AIRFLOW_CTX_TASK_ID='cleanup_temp_files' AIRFLOW_CTX_EXECUTION_DATE='2024-02-11T00:07:16.875125+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-11T00:07:16.875125+00:00'
[2024-02-11T00:08:37.904+0000] {etl_dag.py:252} INFO - Archivo '/tmp/extracted_data.csv' existe antes de la eliminación.
[2024-02-11T00:08:37.909+0000] {etl_dag.py:252} INFO - Archivo '/tmp/transformed_data.csv' existe antes de la eliminación.
[2024-02-11T00:08:37.911+0000] {etl_dag.py:264} INFO - Archivo '/tmp/extracted_data.csv' eliminado con éxito.
[2024-02-11T00:08:37.914+0000] {etl_dag.py:264} INFO - Archivo '/tmp/transformed_data.csv' eliminado con éxito.
[2024-02-11T00:08:37.925+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-11T00:08:38.001+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_weather_data, task_id=cleanup_temp_files, execution_date=20240211T000716, start_date=20240211T000835, end_date=20240211T000838
[2024-02-11T00:08:38.157+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-11T00:08:38.281+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
